src/Bicep.Wasm/Interop.cs

```diff
@@ -116,7 +116,11 @@ private static (string, IEnumerable<dynamic>) CompileInternal(string content)
 
         private static Compilation GetCompilation(string text)
```

Comment:
I wonder if we could somehow setup sample modules in the playground for people to reference.